osgi/jta/src/main/resources/jbossts-properties.xml

```diff
@@ -90,7 +98,6 @@
     -->
     <entry key="RecoveryEnvironmentBean.recoveryModuleClassNames">
         com.arjuna.ats.internal.arjuna.recovery.AtomicActionRecoveryModule
-        com.arjuna.ats.internal.txoj.recovery.TORecoveryModule
```

Comment:
Sure, nps. It is used in some internal Narayana features too but I don't believe it is required when just dealing with XAResources which is what you are likely needing?